script.js

```diff
@@ -82,26 +86,112 @@ searchButton.addEventListener(`click`, event => {
+                    <div class="restaurant-info">
+                        <div class="inside-restaurant-info">
+                            <p class="couisine">Type: ${cuisines}</p>
+                            <p class="cost-for-two">Avg. cost for two: ${average_cost_for_two}</p>
+                            <p class="address">${location.address}</p>
+                            <p class="timings">${timings}</p>
+                        </div>
                     </div>
                 </div>
-            </div>
         `;
+                restaurantsReturned.innerHTML += restaurantNode;
+            });
+        });
+});
 
+searchPrice.addEventListener(`click`, event => {
+    clearResults ()
+    fetch(
+            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418&sort=cost`
+        )
+        .then(repsonse => repsonse.json())
+        .then(({ restaurants }) => {
+            console.log(restaurants);
+            restaurants.map(({ restaurant }) => {
+                const {
+                    name,
+                    featured_image,
+                    cuisines,
+                    average_cost_for_two,
+                    location,
+                    timings,
+                    phone_numbers,
+                    user_rating,
+                    id
+                } = restaurant;
+
+                const restaurantNode = `
+                <div class="restaurant-card" id="restaurant-${id}">
+                <img src="${featured_image}" class="photo" title="Photo of ${name}" />
+                    <div class="content">
+                        <h2 class="card-text card-title">${name}</h2>
+                        <p class="card-text phone">Call on : ${phone_numbers}</p>
+                        <p class="card-text rating">Avg.Rating ${user_rating.aggregate_rating}</p>
+                        <button class="open-more-info button" data-id="restaurant-${id}">View more</button>                </div>
+                    <div class="restaurant-info">
+                        <div class="inside-restaurant-info">
+                            <p class="couisine">Type: ${cuisines}</p>
+                            <p class="cost-for-two">Avg. cost for two: ${average_cost_for_two}</p>
+                            <p class="address">${location.address}</p>
+                            <p class="timings">${timings}</p>
+                        </div>
+                    </div>
+                </div>
+        `;
                 restaurantsReturned.innerHTML += restaurantNode;
             });
         });
+});
+
+searchRating.addEventListener(`click`, event => {
+    clearResults ()
+    fetch(
+            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418&sort=rating`
+        )
+        .then(repsonse => repsonse.json())
```

Comment:
and here as well

Reply:
Ive made the changes to the spelling mistakes